demo/treebank-translation.html

```diff
@@ -2,48 +2,35 @@
-    <div class="alpheios-enabled" id="aligned-lat">
-      <span class="w alpheios-word " data-alpheios_align_word="s2" data-alpheios_align_ref="#aligned-eng span[data-alpheios_align_word='s7']">In</span>
-      <span class="w alpheios-word " data-alpheios_align_word="s3" data-alpheios_align_ref="#aligned-eng span[data-alpheios_align_word='s9']"
-          data-alpheios_tb_ref="phi0959.phi006.alpheios-text-lat1#1-2">nova</span>
-      <span class="w alpheios-word " data-alpheios_align_word="s4"
-        data-alpheios_align_ref="#aligned-eng span[data-alpheios_align_word='s5'],#aligned-eng span[data-alpheios_align_word='s6']">fert</span>
-      <span class="w alpheios-word " data-alpheios_align_word="s5"
-        data-alpheios_align_ref="#aligned-eng span[data-alpheios_align_word='s2']"
-        data-alpheios_tb_ref="phi0959.phi006.alpheios-text-lat1#1-4">animus</span>
-   </div>
-   <div class="alpheios-enabled" id="aligned-eng">
-      <span class="w alpheios-word" data-alpheios_align_word="s1">My</span>
-      <span class="w alpheios-word" data-alpheios_align_word="s2" data-alpheios_align_ref="#aligned-lat span[data-alpheios_align_word='s5']">soul</span>
-      <span class="w alpheios-word" data-alpheios_align_word="s3">is</span>
-      <span class="w alpheios-word" data-alpheios_align_word="s4">wrought</span>
-      <span class="w alpheios-word" data-alpheios_align_word="s5" data-alpheios_align_ref="#aligned-lat span[data-alpheios_align_word='s4']">to</span>
-      <span class="w alpheios-word" data-alpheios_align_word="s6" data-alpheios_align_ref="#aligned-lat span[data-alpheios_align_word='s4']">sing</span>
-      <span class="w alpheios-word" data-alpheios_align_word="s7" data-alpheios_align_ref="#aligned-lat span[data-alpheios_align_word='s1']">of</span>
-      <span class="w alpheios-word" data-alpheios_align_word="s8">forms</span>
-      <span class="w alpheios-word" data-alpheios_align_word="s9" data-alpheios_align_ref="#aligned-lat span[data-alpheios_align_word='s3']">transformed</span>.
-   </div>
+<body data-alpheios_tb_version="3" data-alpheios_tb_src="https://alpheios-project.github.io/treebank-template/embed/DOC/SENTENCE?w=WORD" data-alpheios_tb_ref="on-the-murder-of-eratosthenes-1-50#1-1">
+<script type="text/javascript">
+  document.addEventListener("DOMContentLoaded", function(event) {
+    import ('../dist/alpheios-embedded.js').then(embedLib => {
+      window.AlpheiosEmbed.importDependencies({ mode: 'development' }).then(Embedded => {
+        new Embedded({clientId: 'embed-lib-demo-treebanks', authEnv: null}).activate();
+      }).catch(e => {
+        console.error(`Import of an embedded library dependencies failed: ${e}`)
+      })
+    }).catch(e => {
+      console.error(`Import of an embedded library failed: ${e}`)
+    })
+  });
+</script>
+<div lang="grc" class="alpheios-enabled">
```

Comment:
can you add the translation pieces back?

Reply:
Sure, sorry for overlooking that